CHANGELOG.md

```diff
@@ -19,6 +19,7 @@ The format is based on [Keep a Changelog](http://keepachangelog.com)
 - #2581 - Versioned ClientLibs no longer works with proxied clientlibs
 - #2562 - Fixed cache refresh on versioned clientlibs request when enforceMd5 is false (default).
 - #2590- Fixed issue on 6.4.x with Service User mappings not being registered due to unsupported filename format of OSGi config. 
```

Comment:
Unrelated to this ticket, but might be good to fix the spacing issue here - `#2590 -`